14-location.Rmd

```diff
@@ -215,7 +211,7 @@ reclass
 
 ## Define metropolitan areas
 
-We define metropolitan areas as pixels of 20 km^2^ inhabited by more than 500,000 people.
+We decided to define metropolitan areas as pixels of 20 km^2^ inhabited by more than 500,000 people.
```

Comment:
Does decide add something valuable to the meaning of the sentence -:)? I would prefer the first version but can live with this as well.

Reply:
Maybe it is only me, but I would interpret these two sentences differently:

"We define metropolitan areas as pixels of 20 km^2^ inhabited by more than 500,000 people." (as we ALWAYS define it this way)

"We decided to define metropolitan areas as pixels of 20 km^2^ inhabited by more than 500,000 people." (as we defined it this way for the purpose of the study)

Reply:
Ok, good point, I haven't looked at it this way. So I will rewrite to: "We deliberately define..."